go.mod

```diff
@@ -0,0 +1,13 @@
+module utxodump
+
+go 1.16
```

Comment:
nit: we can bump to 1.18 now (to be consistent across paxosglobal)
```suggestion
go 1.18
```

Reply:
Same as above, though now that I think about it, we should probably take 100% ownership of this